app/src/main/java/org/septa/android/app/rating/RatingUtil.java

```diff
@@ -0,0 +1,117 @@
+package org.septa.android.app.rating;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.design.widget.BottomSheetDialog;
+import android.view.View;
+
+import org.septa.android.app.R;
+
+/**
+ * Rating Utilities
+ */
+
+public class RatingUtil {
+
+    public static final int MIN_USES_TO_RATE = 5; // TODO: change to 20
+
+    // TODO: change to 6 (must be one more than the number of uses)
+    public static final int MIN_CRASH_FREE_USES_TO_RATE = 3; // must be one more than the number of uses wanted
+
+    // increment this when wanting to re-ask user for a rating
+    private static final int CURRENT_RATING_ID = 0;
+
+    private RatingUtil() {
```

Comment:
So it does not appear that this class keep state and all the methods are static.  So just go ahead and make this class Abstract and forget about the constructor.